tests/strategy/test_interface.py

```diff
@@ -106,6 +106,23 @@ def test_get_signal_old_dataframe(default_conf, mocker, caplog, ohlcv_history):
     assert log_has('Outdated history for pair xyz. Last tick is 16 minutes old', caplog)
 
 
+def test_ignore_expired_candle(default_conf, ohlcv_history):
+    default_conf.update({'strategy': 'DefaultStrategy'})
+    strategy = StrategyResolver.load_strategy(default_conf)
+    strategy.ignore_buying_expired_candle = True
+    strategy.ignore_buying_expired_candle_after = 60
```

Comment:
Conditional review comment: If the answer to"should be set in configuration?" from the other comment is true, then you should move these to update the configuration instead of the strategy.

That way, we'll know that the "transfer" from config to strategy works correctly.